ansible/roles/kind/kind_cluster_create/templates/kind.yaml.j2

```diff
@@ -9,8 +9,14 @@ featureGates:
 {% endif %}
 nodes:
  - role: control-plane
+{% if node_image %}
+   image: {{ node_image }}
```

Comment:
change `node_image` to `kind_node_image`, also same in ansible

Reply:
done